torch/distributed/fsdp/_fully_shard/_fsdp_collectives.py

```diff
@@ -600,29 +603,52 @@ def _get_all_gather_input_metadatas(
 
 
 def _get_gradient_divide_factors(
+    factor: Optional[float],
     reduce_scatter_group: dist.ProcessGroup,
     all_reduce_group: Optional[dist.ProcessGroup],
     reduce_dtype: torch.dtype,
     device_type: str = "",
-) -> Union[tuple[None, None], tuple[float, float]]:
+    force_sum_reduction_for_comms: bool = False,
+) -> tuple[
+    Optional[float],
+    Optional[float],
+    Union[dist.ReduceOp, dist.ReduceOp.RedOpType],
+    Union[dist.ReduceOp, dist.ReduceOp.RedOpType],
+]:
+    if device_type == "mtia":
```

Comment:
nit: I know this doesn't change the existing behavior but it would be nice to add context comment as tying device type with `force_sum_reduction_for_comms` (the new var) may not be intuitive to later readers.